webapp/login.go

```diff
@@ -16,6 +17,48 @@ import (
 	"golang.org/x/oauth2"
 )
 
+type loginResponse struct {
+	User  *shared.User `json:"user,omitempty"`
```

Comment:
Note: we should add json annotation to `shared.User`, too; otherwise, it'd have capitalized field names, e.g. `user.GitHubEmail`, which looks inconsistent.